core/dbt/parser/base.py

```diff
@@ -280,9 +279,18 @@ def update_parsed_node_config_dict(
     def update_parsed_node_relation_names(
         self, parsed_node: IntermediateNode, config_dict: Dict[str, Any]
     ) -> None:
-        self._update_node_database(parsed_node, config_dict)
-        self._update_node_schema(parsed_node, config_dict)
-        self._update_node_alias(parsed_node, config_dict)
+
+        if parsed_node.resource_type == "snapshot":
```

Comment:
small nit: let's use the `NodeType.Snapshot` enum here instead if possible

Reply:
Done